docs/use_cases/batch/part1.md

```diff
@@ -198,6 +258,7 @@ CREATE TABLE REGION  (
  }]'
 );
 ```
+</details>
```

Comment:
I would add the following information: 
the schema declared for the tables may not be identical to the schema of the tables stored in S3; for example, you may omit columns that are not used, which can increase efficiency.
I also think we support adding new columns, filled with default values, but we should double-check this works.